CROSS_CHAIN_RECOVERY.md

```diff
@@ -0,0 +1,33 @@
+# Cross Chain Recovery Instructions
+
+In a cross-chain transaction, you attempt to send funds to an address, but accidentally perform the transaction on the wrong chain.
+
+For example, an exchange may ask you to deposit Bitcoin Cash to the Bitcoin Cash address 36jfYqR3J29nAEfYqgksVJ8iVrvLmEb51v. Y  ou misread or don't understand the instructions, and instead send Bitcoin to the address. This is a BTC→BCH cross-chain transaction.
+
+Cross-chain transaction among BTC, LTC, and BCH can be recovered
+
+## What happens in a cross-chain recovery?
+
+Using the same BTC→BCH scenario as above, the goal is to use the private keys of the Bitcoin Cash wallet (that the user was trying to deposit into) to create a wallet on the Bitcoin chain, so that the newly created Bitcoin wallet can claim the transaction sent by the user and return it. Here's how it works:
```

Comment:
... to create a transaction on the Bitcoin Blockchain so that the funds can be recovered.

Reply:
Corrected